providers/azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/domain/AlertProperties.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.azurecompute.arm.domain;
+
+import org.jclouds.javax.annotation.Nullable;
+import org.jclouds.json.SerializedNames;
+
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+public abstract class AlertProperties {
+
+	@Nullable
+	public abstract Object context();
+
+	@Nullable
+	public abstract Object egressConfig();
```

Comment:
Is `Object` the most specific type useful here?

Reply:
Was referring to Type as per the doc https://docs.microsoft.com/en-us/rest/api/monitor/alertsmanagement/alerts/get-all#alertproperties

Reply:
Sorry I'm still confused -- how does the caller use this `Object`?

Reply:
The response seems to be json.
 "egressConfig": {},
  "context": {}
I will change the dataType to JsonBall.  Please let me know your thoughts.